sandbox/grommet-app/src/components/GlobalHeader/GlobalHeader.jsx

```diff
@@ -36,11 +36,12 @@ export const GlobalHeader = ({
   activeTheme,
   backgroundBack,
   setBackgroundBack,
+  ...rest
 }) => {
   const theme = useContext(ThemeContext);
   const breakpoint = useContext(ResponsiveContext);
   return (
-    <Page kind="full">
+    <Page kind="full" {...rest}>
```

Comment:
<img width="1723" alt="Screenshot 2025-01-22 at 1 13 40 PM" src="https://github.com/user-attachments/assets/9e9b2e26-fb1f-4642-a17b-db72ce41f781" />

Should we add z-index to the global header to avoid the elevation being cut off?

Reply:
with updated theme styling this elevation goes away anyway, but for toggling between themes

Reply:
done.